src/store/asana/index.ts

```diff
@@ -404,7 +404,7 @@ export default {
         "&redirect_uri=" +
         location.protocol +
         "//" +
-        location.host +
+        "localhost:3000" +
```

Comment:
Hardcoding this feels like it will cause issues in production? location.host and there should be a location.port as well